### ISSUE TITLE
ldparser needs to be improved for short descriptions containing a '.'

### ISSUE THREAD

**Issue by kartben** [SierraWireless/org.eclipse.koneki.ldt#2]
If the short description contains a dot, the short description is shorter than expected...

e.g. 

``` lua
-------------------------------------------------------------------------------
-- Returns information about an existing battle.net conversation
-- @function [parent=#global] BNGetConversationInfo
-------------------------------------------------------------------------------
```

ends up with "Returns information about an existing battle." for the short description and "net conversation" for the description

**Comment by kartben** [comment]
In LuaDoc, parsing is done this way:

``` lua
-- read until the first period followed by a space or tab
    local summary = string.match(description, "(.-%.)[%s\t]")
```

https://github.com/keplerproject/luadoc/blob/master/src/luadoc/taglet/standard.lua#L115

**Comment by kartben** [comment]
Fixed in fb7afc008fc01a7562fcacd15c14f85eb643b5a9
